tests/test_stats.py

```diff
@@ -235,7 +235,7 @@ def test_energy_bills_report_correctly_market_fees(grid_fees):
     m_bills.update(grid_fees)
     result = m_bills.bills_results
     assert result["street"]["house1"]["market_fee"] == 0.04
-    assert result["street"]["house2"]["market_fee"] == 0.04
+    assert result["street"]["house2"]["market_fee"] == 0.0
```

Comment:
I understand why you had to change this, but I think it would be better if you added one more trade that would change the market_fee of house2. That way you would test that the behavior is consistent for both markets.